nautobot_topology_views/api/views.py

```
@@ -1,37 +1,48 @@
 )
-import netbox_topology_views.models
-from netbox_topology_views.models import RoleImage, IndividualOptions, CoordinateGroup, Coordinate, CircuitCoordinate, PowerPanelCoordinate, PowerFeedCoordinate
-from netbox_topology_views.views import get_topology_data
-from netbox_topology_views.utils import get_image_from_url, export_data_to_xml, get_query_settings
-from netbox_topology_views.filters import DeviceFilterSet
+import nautobot_topology_views.models
+from nautobot_topology_views.models import (
+    RoleImage,
+    IndividualOptions,
+    CoordinateGroup,
+    Coordinate,
```

Ruff (0.8.2)

`nautobot_topology_views.models.Coordinate` imported but unused

Remove unused import

(F401)